.github/workflows/bump-channel-version/action.yml

```diff
@@ -8,6 +8,15 @@ inputs:
   CLEAN_PACKAGE_VERSION:
     required: true
     description: 'The current package version, e.g. "3.11.7".'
+  OVERRIDE_PACKAGE_VERSION:
```

Comment:
Isn't this a hack to make up for the fact that we set the package version elsewhere?

Reply:
I've changed the this is implemented.
I want the pre-release and the release to have different tag names so they won't block each other. 
in order to do so I need two name formats:
release:  `3.16.0-cloud5`
pre-release: ` 3.16.0-cloud5-test` with the postfix.

In order to add the postfix to the package name I need to override it.

Reply:
Not sure this is good - please check what the hosting looks for, it might filter in any version with the string `cloud` in it